capstone-app/src/app/modules/photos/components/menu-item/menu-item.component.ts

```diff
@@ -12,6 +12,39 @@ export class MenuItemComponent {
   @Output() filterSelected = new EventEmitter<string>();
   @ViewChild('childMenu') public childMenu?: MatMenu;
   markerFilter: string;
+  optionsTypes = new Map<string, string>([
```

Comment:
it's possible that we will need to edit this further because I was planning on removing some of the marker type options. For example, I was planning on removing unnecessary ones like casino, church, mosque, store, cafe, shopping mall, etc.

Reply:
Yeah I removed most of the unnecessary ones